### PR TITLE
Pull in Upstream Changes

### PR DESCRIPTION
**Pull request by mcdonnnj**
# 

## 🗣 Description
This PR pulls in the changes in cisagov/skeleton-generic#40 (and a bit more from before).

## 💭 Motivation and Context
Keep it up to date!

## 🧪 Testing
`pre-commit` runs without issue.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Fantastic!

**Review by felddy — APPROVED** [review]
🍪

File: main.tf [hunk 1]
```diff
@@ -7,6 +7,11 @@
 # The AWS account ID being used
 data "aws_caller_identity" "current" {}
 
+# Default provider information.
```
Comment (jsf9k) [inline main.tf]: I suggest moving this provider block above the `aws_caller_identity`, or to a separate file called `providers.tf`.  No big deal if you disagree.

**Review by jsf9k — APPROVED** [review]
Very nice work, @mcdonnnj!  I am following antonbabenko/pre-commit-terraform#100.

I made one suggestion, which you can take or leave.

File: main.tf (hunk 1 above)
Comment (mcdonnnj) [reply main.tf]: Forgot to mention that I fixed this in 0176807 by moving it to `providers.tf`. Thanks for the feedback!

**Review by dav3r — APPROVED** [review]
It's still all good from me. 👍 Thanks for helping us get our Terraform house in order.

**Review by jsf9k — APPROVED** [review]
Still looks good to me!  Approved!

**Pull request merged**